packages/lnd/build.sh

```diff
@@ -6,7 +6,7 @@ TERMUX_PKG_SRCURL=(https://github.com/lightningnetwork/lnd/releases/download/v$T
                    https://github.com/lightningnetwork/lnd/releases/download/v$TERMUX_PKG_VERSION/vendor.tar.gz)
 TERMUX_PKG_SHA256=(ae8cb77eb7567ed9f8041a17eb6f65280cf81f4fbc3bf10eb671a423f62fc948
                    3bbfa000e2b4c7702f92d24235b5a098f37fd7b5830ca42586678f03d7cf9da3)
-TERMUX_PKG_DEPENDS="bitcoin"
+TERMUX_PKG_DEPENDS="bitcoin,termux-services"
```

Comment:
We don't add termux-services as a dependency for the packages that have service scripts, since it's optional to use termux-services. 
We could add termux-services as TERMUX_PKG_SUGGESTS but I don't think that is necessary either.

Reply:
Interesting. I just added it because @xeffyr added it to my earlier bitcoin package. It seems you have different policies.

Reply:
Ah, thanks, didn't see that.

You can also add the run-scripts as conffiles ([as for example emacs does](https://github.com/termux/termux-packages/blob/26785f81ddfb2531c1e5ef5156039dc21e481a3a/packages/emacs/build.sh#L11)) so that the script isn't overwritten on upgrade if the user has changed it

Reply:
Awesome, thanks. Done.